### PR TITLE
Sub multiple field in case data

### PR DESCRIPTION
**Pull request by GauravChawlaMOJ**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ECM-759

### Change description ###

Sub-multiple field was not populating for single cases when multiple excel file was imported. Also batch update for sub multiples wasn't working

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelReadingService.java [hunk 1]
```diff
@@ -76,6 +82,22 @@ public SortedMap<String, Object> readExcel(String userToken, String documentBina
 
     }
 
+    public void setSubMultipleFieldInSingleCaseData(String userToken,
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelReadingService.java:85]: JavaDoc

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleBatchUpdate2Service.java [hunk 2]
```diff
@@ -219,22 +218,31 @@ private void readExcelAndAddSubMultipleRef(String userToken, MultipleDetails mul
...
             MultipleObject multipleObject = (MultipleObject) value;
             if (multipleObjectsFiltered.contains(key)) {
                 multipleObject.setSubMultiple(updatedSubMultipleRef);
+                try {
+                    excelReadingService.setSubMultipleFieldInSingleCaseData(userToken,
+                            multipleDetails,
+                            multipleObject.getEthosCaseRef(),
+                            updatedSubMultipleRef);
+                } catch (IOException e) {
+                    log.error(e.toString());
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleBatchUpdate2Service.java:244]: Maybe log the case number?

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleUploadService.java [hunk 3]
```diff
@@ -79,6 +82,27 @@ public void bulkUploadLogic(String userToken, MultipleDetails multipleDetails, L
...
+                FilterExcelType.ALL);
+        multipleObjects.forEach((key, value) -> {
+            MultipleObject multipleObject = (MultipleObject) value;
+            try {
+                excelReadingService.setSubMultipleFieldInSingleCaseData(userToken,
+                        multipleDetails,
+                        multipleObject.getEthosCaseRef(),
+                        multipleObject.getSubMultiple());
+            } catch (IOException e) {
+                log.error(e.toString());
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleUploadService.java:101]: Maybe log the case number?

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelReadingService.java (hunk 1 above)
Comment (GauravChawlaMOJ) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelReadingService.java:92]: added

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleBatchUpdate2Service.java (hunk 2 above)
Comment (GauravChawlaMOJ) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleBatchUpdate2Service.java]: added case number in log

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleUploadService.java (hunk 3 above)
Comment (GauravChawlaMOJ) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleUploadService.java]: added case number in log

**Pull request merged**